CMakeLists.txt

```diff
@@ -399,6 +400,7 @@ if(BUILD_TEST)
     ${NVFUSER_ROOT}/test/test_tutorial.cpp
     ${NVFUSER_ROOT}/test/test_alias_analysis.cpp
     ${NVFUSER_ROOT}/test/test_scalar_hoisting.cpp
+    ${NVFUSER_ROOT}/test/test_no_op.cpp
```

Comment:
lol. did we forget to build the test the whole time?!

Reply:
Fortunately, not the whole time. My recent #1205 missed that...